README.md

```diff
@@ -107,7 +108,7 @@ Once the user has logged into your app, inform Streem that they are logged in:
     }
 ```
 
-There are two representations of the user. The user in your system, and the user in Streem's system. Streem uses the userId as an identifier to a user in our system, and `userId` is intended to stay consistent per user in your system. The properties of `expert`, `name`, and `avatarUrl`, all of which are supplied by you in this identify call, are per user via that `userId`. Changing these values in subsequent identify calls will update their values in Streem's system.
+The `companyCode` is a code that streem will provide your company with; it is the url prefix when using the Steem Web App (`https://{company-code}.swa.dev.streem.cloud`). The `email` and `password` fields are the credentials a user has set up through the Streem Web App. `expert` indicates whether or not the user trying to login is an agent or a customer. This value defaults to `true`. `avatarUrl` is an optional string containing the url for a profile picture the user would like to be visible during streems.
```

Comment:
`that Streem will provide`

Reply:
[✅](https://github.com/streem/streem-sdk-ios/pull/16/commits/9a9971e525117729bee454e3726cce8268421c07)